cli/commons/src/Microsoft.Kiota.Cli.Commons/IO/JsonOutputFormatterOptions.cs

```diff
@@ -3,11 +3,16 @@
 /// <summary>
 /// The JSON output formatter options
 /// </summary>
-public class JsonOutputFormatterOptions : OutputFormatterOptions
+public class JsonOutputFormatterOptions : IJsonOutputFormatterOptions
 {
     /// <summary>
-    /// Gets or sets a value that defines whether JSON should use pretty printing. By
-    /// default, JSON is serialized without any extra white space.
+    /// Create new instance of JSON output formatter
     /// </summary>
-    public bool OutputIndented { get; set; } = true;
+    public JsonOutputFormatterOptions(bool outputIndented)
```

Comment:
```suggestion
    public JsonOutputFormatterOptions(bool outputIndented = true)
```
The default constructor may be useful

Reply:
Why? I don't expect users to actually create an instance of this class directly. They'd instead call `FormatterType.GetOutputFormatterOptions`